android/src/com/google/zxing/client/android/wifi/WifiConfigManager.java

```diff
@@ -17,6 +17,7 @@
 package com.google.zxing.client.android.wifi;
 
 import android.net.wifi.WifiConfiguration;
+import android.net.wifi.WifiEnterpriseConfig;
```

Comment:
I think the reason this had been rejected before is that it requires API 18. I think that's OK to require at this point. But it would require updating the Manifest and build files to reflect that minimum too.

Reply:
So I should update https://github.com/zxing/zxing/blob/87f00d41a747382a1dc34a6e67bfb3b21c4c4236/android/AndroidManifest.xml#L34 this line to say "android:minSdkVersion="18""?